cypress/support/utils/packing_instructions_version.js

```diff
@@ -31,18 +37,22 @@ export class PackingInstructionsVersion {
 function applyPackingInstructionsVersion(packingInstructionsVersion) {
   describe('create new packing instructions version', function() {
     cy.visitWindow('540344', 'NEW');
-    cy.selectInListField('M_HU_PI_ID', packingInstructionsVersion.packingInstructions);
+    cy.selectInListField('M_HU_PI_ID', packingInstructionsVersion.packingInstructions, false, null, true);
     cy.writeIntoStringField('Name', packingInstructionsVersion.name);
     cy.clickOnCheckBox('IsCurrent');
+    if (packingInstructionsVersion.unit) {
+      cy.selectInListField('HU_UnitType', packingInstructionsVersion.unit, false, null, true);
```

Comment:
won't this work just with `cy.selectInListField('HU_UnitType', packingInstructionsVersion.unit);`? (note, i haven't tested, just asking)

Reply:
On my machine it doesn't without.

Reply:
What if you use `cy.resetListValue()` before setting in the list field? I have the feeling that's the problem now

Reply:
done.